File: examples/python/get_vehicle_details.py
import sys
from pprint import pprint
from apibara_client import apibara_get

vin_or_lot = sys.argv[1] if len(sys.argv) > 1 else "WBAVL1C58FVY28848"
vehicle = apibara_get(f"/vehicles/{vin_or_lot}")

pprint(vehicle)

File: examples/typescript/get-vehicle-details.ts
import { apibaraGet } from "./apibara-client.js";

const vinOrLot = process.argv[2] || "WBAVL1C58FVY28848";
const data = await apibaraGet(`/vehicles/${encodeURIComponent(vinOrLot)}`);

console.log(JSON.stringify(data, null, 2));
